android/app-example/src/main/java/com/badoo/ribs/example/app/OtherActivity.kt

```diff
@@ -0,0 +1,33 @@
+package com.badoo.ribs.example.app
+
+import android.app.Activity
+import android.content.Intent
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity;
+import android.widget.TextView
+import com.badoo.ribs.example.R
+
+import kotlinx.android.synthetic.main.activity_other.*
+
+class OtherActivity : AppCompatActivity() {
```

Comment:
When clicking the FAB button and goes back to the previous screen, there is no view presented.
However the tab Hello is selected.

Reply:
Created issue #9, will fix in separate branch.